config/flags.go

```diff
@@ -8,6 +8,11 @@ import (
 	"github.com/urfave/cli/v2"
 )
 
+// Env vars
+var (
+	HealthBlockTimeout = "BLOCK_TIMEOUT"
```

Comment:
Why you decide to use ENV variable since most of variables are flag one? Just curios that all of configs related to metrics are set with flags...

Reply:
I don't expect this to be a permanent solution. We should probably have it as an optional field for each chain in the config file. Env var just seems less intrusive